src/NServiceBus.Persistence.NonDurable.AcceptanceTests/When_setting_timetokeepdeduplicationdata.cs

```diff
@@ -94,16 +91,14 @@ public OutboxEndpoint()
                     // limit to one to avoid race conditions on dispatch and this allows us to reliably check whether deduplication happens properly
                     b.LimitMessageProcessingConcurrencyTo(1);
                     b.EnableOutbox().TimeToKeepDeduplicationData(TimeSpan.FromSeconds(3));
-                    b.GetSettings().Set("Outbox.NonDurableTimeToCheckForDuplicateEntries", TimeSpan.FromSeconds(1));
+                    b.GetSettings().Set("Outbox.NonDurableTimeToCheckForDuplicateEntries", TimeSpan.FromMilliseconds(100));
```

Comment:
Why was this needed? The delay is 5 seconds in the test.

Reply:
This is how often we should check, not how long we keep them